site/tutorials/streaming.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: tutorials
+menu: tutorials
+title: Streaming Data
+permalink: /tutorials/streaming.html
+---
+
+
+In this tutorial, you'll learn how to stream new data in a Vega-Lite visualization. If you are not familiar with Vega-Lite, please read the [getting started tutorial]({{site.baseurl}}/tutorials/getting_started.html) first.
+
+We will be using the `View` API from the Vega, where we update data via the [`change`](https://vega.github.io/vega/docs/api/view/#data) method.  Data in Vega and Vega-Lite is updated either via interactions, or by specifying the data changes directly.  In both cases, the data is pushed through the Vega runtime for optimized performance.  This tutorial focuses on the latter for streaming data, where no user interaction is involved.
+
+<div id="chart" class="vl-example"></div>
```

Comment:
i wanted to inline the visualization for better formatting (like in explore data tutorial) but didn't locate the css that did it; and float left looked weird. I'll remove the vl-example (forgot to afterwards)